src/shared/managers/SdkEnvironment.ts

```diff
@@ -92,13 +92,13 @@ export default class SdkEnvironment {
     switch (buildEnv) {
       case EnvironmentKind.Development:
         if (SdkEnvironment.isTurbineEndpoint(action)) {
-          return new URL(`http://${apiOrigin}:${TURBINE_API_URL_PORT}/api/v1`);
+          return new URL(`http://${apiOrigin}:${TURBINE_API_URL_PORT}/api/v1/`);
         }
-        return new URL(`http://${apiOrigin}:${API_URL_PORT}/api/v1`);
+        return new URL(`http://${apiOrigin}:${API_URL_PORT}/api/v1/`);
       case EnvironmentKind.Staging:
-        return new URL(`https://${apiOrigin}/api/v1`);
+        return new URL(`https://${apiOrigin}/api/v1/`);
       case EnvironmentKind.Production:
-        return new URL('https://onesignal.com/api/v1');
+        return new URL('https://api.onesignal.com/');
```

Comment:
This is used in a few places where this would be a breaking change:
1. subscribeSafariPromptPermission() - This URL has to match exactly with the .zip files generated for macOS Safari legacy push support. I'll include the full internal details in the Asana ticket.
2. getNotificationIcons() - This fetches a icon, so api.onesignal.com may not work. If it does work, I still don't think it should be used that way, unless someone from the backend can confirm this won't break in the future.